opendrift/readers/basereader/variables.py

```diff
@@ -578,6 +546,7 @@ def get_variables_interpolated_xy(self,
 
         x = np.atleast_1d(x)
         y = np.atleast_1d(y)
+        z = np.atleast_1d(z) if z is not None else np.zeros((1,))
```

Comment:
@knutfrode : dette kan potensielt lage trøbbel. Men akuratt no krasjar convers_position hvis z er satt til None.. alternativt kan denne fiksen gjerast i covers_positions. Men lenger nede sjekkast det og for lengde av z, so det må håndterast der og.